build.sbt

```diff
@@ -234,6 +237,18 @@ lazy val uzhttp = project
   )
   .dependsOn(core)
 
+lazy val play = project
+  .in(file("adapters/play"))
+  .settings(name := "caliban-play")
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.typesafe.play" %% "play"                % playVersion,
+      "de.heikoseeberger" %% "akka-http-play-json" % "1.32.0"
```

Comment:
Why do we need the dependency to `akka-http-play-json`?

Reply:
It's because I'm using [PlayJsonBackend](https://github.com/ghostdogpr/caliban/blob/master/adapters/akka-http/src/main/scala/caliban/interop/play/PlayJsonBackend.scala) from AkkaHttp module, Maybe we just copy/paste this code to avoid that dependency?

Reply:
Can it be written without the Akka stuff as a pure play thing? If that’s the case I think it’s better to avoid the dependency.

Reply:
Yes! I've written it already (you can check it out in the last commit). The only thing I'm not pretty sure if it should be public for the users o not. What do you think?